src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/IHtmlHelper.cs

```diff
@@ -133,7 +135,7 @@ public interface IHtmlHelper
             string controllerName,
             object routeValues,
             FormMethod method,
-            bool suppressAntiforgery,
+            bool? suppressAntiforgery,
```

Comment:
Put another way: What is the use case for forcing an antiforgery token?

Name doesn't make much sense if we think there's a real scenario here for `null` as distinct from `false` either. Even with IntelliSense help, hard to read `suppressAntiforgery: false` as "force addition of an antiforgery token": User just asked for token not to be _suppressed_. So would have to reverse the polarity and name it `includeAntiforgery` with `null` meaning "default", `false` meaning "no", and `true` meaning "yes".